rethinkdb_update_block.py

```diff
@@ -0,0 +1,84 @@
+from nio.signal.base import Signal
+from nio.properties import (StringProperty, ListProperty, PropertyHolder,
+                            Property)
+from nio.util.discovery import discoverable
+from .rethinkdb_base_block import RethinkDBBase
+
+
+class MatchItem(PropertyHolder):
+    key = Property(title='Match key')
+
+
+@discoverable
+class RethinkDBUpdate(RethinkDBBase):
+
+    """a block for updating info in a RethinkDB table"""
+
+    table = StringProperty(title="Table to update", default='test',
+                           allow_none=False)
+    filters = ListProperty(MatchItem, title='Match the following document keys',
+                           default=[])
+
+    def __init__(self):
+        super().__init__()
+        # current table being updated
+        self._table = None
+
+    def configure(self, context):
+        super().configure(context)
+        self._set_table()
+
+    def process_signals(self, signals):
+        output_signals = []
+        for signal in signals:
+            self.logger.debug('Update is Processing signal: {}'.format(signal))
+            output_signals.append(Signal(self.update_table(signal)))
```

Comment:
I think we should be updating the input signals with the results of the table update. Not making new signals to notify.